example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -24,8 +22,7 @@ public boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-          new RNSharePackage()
```

Comment:
I didn't see a bump to the new auto-linking react-native-cli, but the linking seems to be removed, is the example still a functioning react-native-share demonstration? 🤔

Reply:
I was testing and i cant run the example with the master version. This PR is now broken because i was testing a wat to import the library from a diff folder.

Reply:
ah, okay. clearly labeled WIP so makes sense :)